### PR TITLE
chore: remove curve as source

### PR DESCRIPTION
**Pull request by cehali**

### CODE REVIEW

**Review by rpiszczatowski — APPROVED** [review]

File: packages/oracle-node/manifests/data-services/primary.json [hunk 1]
```diff
@@ -139,24 +139,14 @@
         "uniswap-v3-on-chain-weth-500"
       ]
     },
-    "FRXETH": {
-      "source": ["curve-eth", "uniswap-v3-on-chain-weth-500"]
```
Comment (rpiszczatowski) [inline packages/oracle-node/manifests/data-services/primary.json:143]: Why remove also the uni source?

File: packages/oracle-node/manifests/data-services/primary.json [hunk 2]
```diff
@@ -44,6 +44,8 @@
         "coinbaseprime",
         "coinbasepro",
         "coingecko",
+        "curve-usdc",
```
Comment (rpiszczatowski) [inline packages/oracle-node/manifests/data-services/primary.json:47]: Why not remove this?

**Review by rpiszczatowski — APPROVED** [review]

File: packages/oracle-node/manifests/data-services/primary.json (hunk 1 above)
Comment (cehali) [reply packages/oracle-node/manifests/data-services/primary.json:143]: This token was used only in `curve-frxeth` for ALETH which is not used anymore so `FRXETH` is not necessary

File: packages/oracle-node/manifests/data-services/primary.json (hunk 2 above)
Comment (cehali) [reply packages/oracle-node/manifests/data-services/primary.json:47]: Those pools still have a lot of liquidity, we will also monitor them. This is why I decided to leave them

**Pull request merged**